src/components/Accordion/Accordion.js

```diff
@@ -22,7 +22,7 @@ const Accordion = ({ className, sections, ...props }) => {
       className={classNames(className, "p-accordion")}
       {...props}
       role="tablist"
-      aria-multiselect="true" //eslint-disable-line jsx-a11y/aria-props
+      ariaMultiselect="true" //eslint-disable-line jsx-a11y/aria-props
```

Comment:
Did you mean to use `area-multiselectable` ? 

https://www.w3.org/WAI/PF/aria-1.1/states_and_properties#aria-multiselectable